scifight/migrations/0001_initial.py

```diff
@@ -28,14 +29,15 @@ class Migration(migrations.Migration):
                 ('fight_num', models.IntegerField()),
                 ('start_time', models.DateTimeField(blank=True, null=True)),
                 ('stop_time', models.DateTimeField(blank=True, null=True)),
-                ('status', models.PositiveSmallIntegerField(default=0)),
+                ('status', models.PositiveSmallIntegerField(choices=[(0, 'Not started'), (1, 'In progress'), (2, 'Completed')], default=0)),
```

Comment:
Ah, fuck, you've recreated the file from scratch.
How about naming this file like `0001_initial_2.py` to prevent Git from showing this *meaningless* and *misleading* diff?